pylti/tests/test_flask.py

```diff
@@ -467,19 +494,23 @@ def test_access_to_oauth_resource_post_grade2_fail(self):
                u'008437924c9852377e8994829aaac7a1')
 
         def request_callback(request, cburi, headers):
```

Comment:
This function is duplicated many times over. Why not extract it out at the top of the file?